packages/algo-ts/src/primitives.ts

```diff
@@ -4,7 +4,7 @@ import { BigUintCls, BytesCls, getNumber, Uint64Cls } from './impl/primitives'
 export type Uint64Compat = uint64 | bigint | boolean | number
 export type BigUintCompat = bigint | bytes | number | boolean
 export type StringCompat = string
-export type BytesCompat = bytes | string
+export type BytesCompat = bigint | bytes | string
```

Comment:
I'm a bit unsure about this one. It feels ambiguous to start passing numeric values to args that expect bytes

Reply:
It was used for abi method to `biguint` value as `bytes`. I think we can also achieve this by adding `toBytes` function to `biguint` type. 
Example code in Python below.
```python
@arc4.abimethod()
def verify_biguint_add(self, a: Bytes, b: Bytes) -> Bytes:
    a_biguint = BigUInt.from_bytes(a)
    b_biguint = BigUInt.from_bytes(b)
    result = a_biguint + b_biguint
    return result.bytes
```

Reply:
I see the issue now. We do not need to change `BytesCompat` type for the above requirement. `Bytes(v: biguint)` already exists. It has been reverted. 20b1567